sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -7224,8 +7224,8 @@ def test_pretty_misc_functions():
     assert upretty(fresnelc(x)) == 'C(x)'
     assert pretty(fresnels(x)) == 'S(x)'
     assert upretty(fresnels(x)) == 'S(x)'
-    assert pretty(Heaviside(x)) == 'Heaviside(x)'
-    assert upretty(Heaviside(x)) == 'θ(x)'
+    assert pretty(Heaviside(x)) == 'Heaviside(x, 1/2)'
+    assert upretty(Heaviside(x)) == 'θ(x, 1/2)'
```

Comment:
I wonder if the extra arg should be omitted in the printing for the default value of `1/2`.

Reply:
That would be a very good idea, but I do not know where to do it. Would I need to write a `def _print_Function` in the `class Heaviside`?

Reply:
It's the other way round. You add a `_print_Heaviside` method to the pretty printer:
https://github.com/sympy/sympy/blob/92e7cb0b27bb810d67013a2cdbbd1f8e7f7aba45/sympy/printing/pretty/pretty.py#L1574-L1576

Reply:
OK, I did it ;)  Push comes as soon as I have run all checks I think I need to run locally.

Reply:
I wondered whether we should also change the normal printer, but I think we should just leave it as it is, such that people using this will see immediately that `Heaviside` has changed. Is that OK?

Reply:
What do you mean by the normal printer? SymPy has so many printers...

Reply:
The printer that activates for in-line documentation and is tested through bin/doctest. I don't actually know which one that is ;)